include/baresip.h

```diff
@@ -219,6 +222,7 @@ int  call_answer(struct call *call, uint16_t scode, enum vidmode vmode);
 int  call_progress_dir(struct call *call,
 		       enum sdp_dir adir, enum sdp_dir vdir);
 int  call_progress(struct call *call);
+void call_reject(struct call *call, uint16_t scode, const char *reason);
```

Comment:
is this the same as call_hangup() ?

Reply:
No, there is a big difference.

`call_hangup()` needs already a SIP session `call->sess` to send the BYE.

`call_reject()` marks the call as REJECTED, which triggers in `call_arrived()` a SIP reply with `sip_treply()`. The status code and reason can be specified by the application (menu, ...). The SIP session is not allocated. Also no streams are allocated.